Formula/r/renovate.rb

```diff
@@ -37,5 +37,8 @@ def install
 
   test do
     assert_match "FATAL: You must configure a GitHub token", shell_output("#{bin}/renovate 2>&1", 1)
+
+    expected_output = shell_output("#{bin}/renovate --platform=local --enabled=false 2>&1")
+    assert_match(/^(?!.*ERROR: Unsupported node environment detected\.).*$/, expected_output)
```

Comment:
Since it'll fail otherwise we don't need to check the output.
```suggestion
    system bin/"renovate, "--platform=local", "--enabled=false"
```

Reply:
You're correct. I suggest removing the existing test then, which appears to be a simple check for command executability. The proposed change already covers this.